src/sales/credit-memo-create.md

```diff
@@ -184,56 +200,56 @@ Credit memos include the following sections.
 |Discount Amount|Any discount that applies to the returned item.|
 |Row Total|The line item total, including applicable taxes that are due for the returned product level, less discounts.|
+|order total||
 
-### Credit Memo History
+### Credit Memo Comments
 
 |--- |--- |
 |Comment Text|A text box that is used to enter a comment to the customer about the credit memo.|
 
-### Credit Memo Totals
+### Refund Totals
 
 |--- |--- |
-|Subtotal|Total amount without shipping, tax, or adjustments.|
-|Shipping & Handling|The shipping amount to be refunded.|
+|Refund Shipping|The shipping amount to be refunded.|
 |Adjustment Refund|An amount that is added to the total amount refunded as an additional refund that does not apply to any particular part of the order, such as shipping, items, or tax. The amount entered cannot raise the total refund higher than the amount paid.|
 |Adjustment Fee|An amount that is subtracted from the total amount refunded, such as a restocking fee, or an amount that is related to gift wrapping or gift options.|
 |Grand Total|The total amount to be refunded|
-|Tax|Amount of tax to be refunded.|
-|<span class="ee-only">Store Credit</span>|Amount applied to store credit.|
 |Append Comments|Checkbox that determines if comments are included in the credit memo.|
 |Email Copy of Credit Memo|Checkbox that determines if a copy of the credit memo is emailed.|
 |<span class="ee-only">Refund to Store Credit</span>|Checkbox that determines if the total is to be refunded to [store credit]({% link sales/store-credit-using.md %}).|
 |<span class="b2b-only">Subtotal</span>|The total of all line items to be refunded.|
 {: .buttons-table }
 
-### Refund buttons
+### Refund Buttons
 
-The payment method used for the order determines the refund buttons that are available at the top of a credit memo page.
+The payment method used for the order determines the refund buttons that are available for a credit memo.
 
 |--- |--- |
 |<span class="btn">Refund</span>|If the original purchase was paid by credit card through a payment gateway, the refund amount is managed by the payment processor. To manage refunds, see the documentation provided by your payment provider.|
 |<span class="btn">Refund Offline</span>|If the original purchase was paid by check or money order, the refund is paid directly to the customer, by issuing a check, gift card, or cash if you have a brick and mortar storefront. The credit memo serves as a record of the offline transaction.|
-|<span class="b2b-only"></span><span class="btn">Refund to Company Credit</span>|If the purchase was charged to company credit, the refund is returned to the [company account]({% link customers/account-dashboard-company-credit.md %}).|
+|<span class="b2b-only"></span><span class="btn">Refund to Company Credit</span>|If the purchase was charged to company credit, the refund is returned to the [company’s account]({% link customers/account-dashboard-company-credit.md %}).|
```

Comment:
Best practice to avoid non-person possessives. We and it is titled as Company Account in the UI.